sdk/core/azure-core/azure/core/messaging.py

```diff
@@ -193,8 +187,7 @@ def from_dict(cls, event):
         return event_obj
 
     @classmethod
-    def from_json(cls, event):
-        # type: (Any) -> CloudEvent
+    def from_json(cls, event: Any) -> "CloudEvent":
```

Comment:
I was wondering if there was a specific JSON type or alias and came across this [thread] (https://github.com/python/typing/issues/182) with a lot of discussion w/ pros and cons of various solutions. More as an  FYI

Reply:
Thanks for the link! Interesting read :)